muon/browser/muon_browser_process_impl.cc

```diff
@@ -6,27 +6,56 @@
 
 #include "atom/browser/api/atom_api_app.h"
 #include "atom/browser/atom_resource_dispatcher_host_delegate.h"
+#include "base/path_service.h"
 #include "brave/browser/component_updater/brave_component_updater_configurator.h"
 #include "chrome/browser/chrome_device_client.h"
 #include "chrome/browser/io_thread.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/profiles/profile_manager.h"
+#include "chrome/browser/safe_browsing/safe_browsing_service.h"
 #include "components/component_updater/component_updater_service.h"
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/resource_dispatcher_host.h"
 
 MuonBrowserProcessImpl::MuonBrowserProcessImpl(
       base::SequencedTaskRunner* local_state_task_runner) :
-    BrowserProcessImpl(local_state_task_runner) {
+    BrowserProcessImpl(local_state_task_runner),
+    created_safe_browsing_service_(false) {
   g_browser_process = this;
 
   device_client_.reset(new ChromeDeviceClient);
 }
 
 MuonBrowserProcessImpl::~MuonBrowserProcessImpl() {
+  if (safe_browsing_service_.get())
+    safe_browsing_service()->ShutDown();
   g_browser_process = NULL;
 }
 
+safe_browsing::SafeBrowsingService*
```

Comment:
these should go in chromium_src browser_process_impl
muon_browser_process_impl is for muon extensions that aren't part of chrome browser_process_impl